hw/drivers/sensors/icp101xx/syscfg.yml

```diff
@@ -0,0 +1,52 @@
+# The BSD License (BSD)
```

Comment:
The Adafruit copyright license should not be used as Sterling suggested.

Reply:
Adafruit copyright is fine, it's the proprietary license that shouldn't be used.

Reply:
The Adafruit license was copied from another driver that we have as example. Could we put instead the Invensence license ?

Reply:
the invense license is the problem!  the adafruit license is fine.  I posted a link above for acceptable third party licenses -- 3-clause BSD is OK, however Invensense's proprietary license is not.